lib/DatabaseManager.js

```diff
@@ -102,13 +102,13 @@ DatabaseManager.prototype.populateDb = function(populatePathPattern) {
     .filterModule(_.isFunction)
     .require();
 
-  return Promise.all(
-    _.map(modules, function(module) {
+  return _.reduce(modules, function(res, module) {
+    return res.then(() => knex.transaction(function(trx) {
```

Comment:
That looks weird of having 2 nested transactions here. I don't think that is needed.

Reply:
Oops, that's a mistake while rebasing, I'll fix that and check tests.